### DISCUSSION TITLE
filter changes in a PR by team using CODEOWNERS file

### DISCUSSION THREAD

**Discussion by csmith0651** [community/community#10622] [Pull Requests]
It would be really great if we could use the CODEOWNERS file in conjunction with teams on github to filter PRs files/code changes by team.

We make PRs of all the changes since the last release and it is just too many changes to reasonable review. However, if I could filter the PR down to just my team then I would have more confidence in my review.

Thanks!

Craig

**Comment by seannyphoenix-qz** [comment]
I have come across this recently with a slightly different angle - I want to see what files are owned by a different team so I can inform that team of what changes they own. Being able to filter files by any CODEOWNER, including a GitHub team, would be very useful.
